src/Template/PhpTemplate.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+namespace Dhii\Output\Template;
+
+use ArrayAccess;
+use Dhii\Output\PhpEvaluator\PhpEvaluatorInterface;
+use Dhii\Output\Template\PhpTemplate\Exception\RendererException;
+use Dhii\Output\Template\PhpTemplate\Exception\TemplateRenderException;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception;
+use InvalidArgumentException;
+use OutOfRangeException;
+use Psr\Container\ContainerInterface;
+
+/**
+ * A template that uses PHP code as an engine.
+ */
+class PhpTemplate implements TemplateInterface
+{
+    const VAR_CONTEXT = 'c';
+    const VAR_FUNCTION = 'f';
+
+    /**
+     * @var \Dhii\Output\PhpEvaluator\PhpEvaluatorInterface
+     */
+    protected $evaluator;
+    /**
+     * @var array<string, mixed>
+     */
+    private $defaultContext;
+    /**
+     * @var array<string, callable>
+     */
+    private $functions;
```

Comment:
What is this blasphemy?

Reply:
Just settings of PHPStorm auto-generation. Will fix, thanks :)